tests/test_service.py

```diff
@@ -390,6 +393,21 @@ def testGetItems(self, mockGet):
         self.assertIn((os.path.join(endpoint, "dashboards"),), tuple(mockGet.call_args))
         self.assertEquals(len(mockGet.call_args_list), 1)
 
+class TestGroupPermissions(TestServiceBase):
+    @mock.patch('requests.Session.get', return_value=MockResponse(json.dumps(groupPermission_D), 200))
+    def testGroupPermissionsID(self, mockPost):
+        res = self.argus.grouppermissions.get_permissions_for_group(permissionGroupId)
+        self.assertEquals(res.get("permissionIds"), [0, 1, 2])
+        self.assertIn((os.path.join(endpoint, "grouppermission"),), tuple(mockPost.call_args))
+
+    @mock.patch('requests.Session.post', return_value=MockResponse(json.dumps({testId: [groupPermission_D]}),200))
+
+    def testGetPermissions(self, mockPost):
+        resp = self.argus.grouppermissions.get_permissions_for_group(testId)
+        for id, perms in resp.items():
+            for p in perms:
+                self.assertTrue(isinstance(p, Permission))
```

Comment:
Shouldn't Permission be a GroupPermission?

Reply:
just realised my latest changes werent pushed from local, sorry about that!